samples/gui/lvgl/nrf52840_pca10056.overlay

```diff
@@ -1,11 +1,15 @@
 /*
  * Copyright (c) 2018 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+ * Copyright (c) 2019 O.S.Systems
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 &spi1 {
 	status = "okay";
+	sck-pin = <47>;
+	mosi-pin = <45>;
+	miso-pin = <46>;
```

Comment:
@chtavares592 Could you also fix `spi-max-frequency` and set it to `spi-max-frequency = <10000000>;`?

Reply:
The `spi-max-frequency` is set according the ili datasheet and the nrf spi driver will use the closest supported frequency that is lower then the max frequency.

Reply:
okay, then maybe `spi-max-frequency = <15000000>;`?  `15151515` looks like a typo.

Reply:
I just kept the same value but I tested it with 10000000 and it works too. If is necessary I can include this change on that commit.

Reply:
@jfischer-phytec-iot , if you believe this is really a typo, then @chtavares592 could just update the value, IMO, as part of this PR.

Reply:
> 15151515 looks like a typo.

Sorry EE background, just converted the timing requirements and then you end up 15.15... MHz

Btw no problem with another value (eg 15000000) if it makes it more clear in the end the SPI driver will pick the actual speed

Reply:
So, Am I need to change the `spi-max-frequency` value?